functions/src/index.ts

```diff
@@ -119,7 +123,11 @@ async function handleUpdateDocument(
   try {
     logs.updateDocument(documentId)
     if (validateDocumentId(documentId)) {
-      const document = adaptDocument(documentId, after)
+      const document = adaptDocument(
+        documentId,
+        after,
+        config.meilisearch.fieldsToIndex || ''
```

Comment:
Same about:
```js
    fieldsToIndex: process.env.MEILISEARCH_FIELDS_TO_INDEX || '',
```
Is it not the same parameter?

Reply:
because the type of `MeilisearchConfig` specifies that it can be an undefined

```js
export type MeilisearchConfig = {
  host: string
  apiKey: string
  indexUid: string
  fieldsToIndex?: string
}
```

And typescript does not infer that the variable was already set to string in config so it raises an error